pom.xml

```diff
@@ -128,6 +134,11 @@
        <version>1.3</version>
        <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
This is very rarely needed. Guava can do almost everything this can do. You only seem to use this in one place, so set scope to test if you must use it.

Reply:
removed.